random.json

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
Can we rename this file to providers.json as random.json doesn't mean a great deal!

Reply:
Yes, sure. I've updated the filename to providers.json.